packages/fuel-gauge/src/contract.test.ts

```diff
@@ -699,10 +699,10 @@ describe('Contract', () => {
   });
 
   test('Read only call', async () => {
-    // #region typedoc:Contract-read-only-call
+    // #region Contract-read-only-call
     const contract = await setupContract();
     const { value } = await contract.functions.echo_b256(contract.id.toB256()).get();
     expect(value).toEqual(contract.id.toB256());
-    // #endregion
+    // #endregion Contract-read-only-call
```

Comment:
The Vitepress native import script plugin works this way so when implementing a custom one this was one thing that I did not change.

Reply:
@camsjams I think the main reason behind it is because it supports inner scripts.

![Screenshot from 2023-03-20 20-54-31](https://user-images.githubusercontent.com/30977845/226490200-097148a0-1ad4-471a-aa38-e55a14c714dc.png)
![Screenshot from 2023-03-20 20-32-03](https://user-images.githubusercontent.com/30977845/226490206-f99f186c-e133-4758-a997-95a31a4c4130.png)